src/nav/browser.yml

```diff
@@ -1,66 +1,98 @@
 title: Browser monitoring
 path: /docs/browser
 pages:
-  - title: Introduction to browser monitoring
+  - title: Intro to browser monitoring
     path: /docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring
-  - title: Compatibility and requirements
-    path: /docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring
-  - title: Browser agent EOL policy
-    path: /docs/browser/browser-monitoring/getting-started/browser-agent-eol-policy
-  - title: Browser agent release notes
-    path: /docs/browser/new-relic-browser/getting-started/browser-agent-release-notes
-  - title: Guides
+  - title: Single page app (SPA) monitoring
```

Comment:
@paperclypse It's not within scope, but I'm planning on merging all these SPA docs into the primary browser docs. For now though, I think they're going to be weirdly out of place.

Reply:
@a-sassman Works for me! I think it's fine in the short term.